app/(main)/login/page.tsx

```diff
@@ -82,10 +82,17 @@ const Login = (): React.JSX.Element => {
    * Handles the form submission.
    * @param {LoginUserSchemaType} data - The data from the form.
    */
-  const onSubmit: SubmitHandler<LoginUserSchemaType> = async (data) => {
-    setIsLoading(true);
-    await login(data);
-    setIsLoading(false);
+  const onSubmit: SubmitHandler<LoginUserSchemaType> = async (
+    data: LoginUserSchemaType,
+  ): Promise<void> => {
+    try {
+      setIsLoading(true);
+      await login(data);
+      setIsLoading(false);
+    } catch (error) {
+      setIsLoading(false);
+      console.error('Login Failed', error);
+    }
```

Comment:
Should this `throw` an error instead of console.error?

Reply:
I think so, I'll fix that in my next commit; good catch!

Reply:
Updated in latest commit.